src/main/resources/.data/carpet-redcraft-addons/scripts/armorstandeditor.sc

```diff
@@ -487,3 +487,18 @@ _list_poses() -> (
     );
     _sound(player(), pos(player()), 'click');
 );
+
+// PARTICLES
+global_tick = 0;
+__on_tick() -> (
+    if(global_tick == 20,
+    for(filter(entity_list('*'), _ == 'Armor Stand'),
+        position = str(pos(_):0 + ' ' + str(pos(_):1 + 0.5) + ' ' + pos(_):2);
+        if (query(_, 'has_scoreboard_tag', 'gb.SmokeParticles'), run(str('particle minecraft:campfire_signal_smoke ' + position + ' .2 1 .3 .05 0')),
+            query(_, 'has_scoreboard_tag', 'gb.FlameParticles'), run(str('particle minecraft:flame ' + position + ' .01 .01 .01 .005 2'))
```

Comment:
Suggerimento: cambiare la direzione delle particelle in funzione della rotazione del player